internal/pkg/service/buffer/store/export.go

```diff
@@ -49,6 +49,22 @@ func (s *Store) createExportBaseOp(_ context.Context, export model.ExportBase) o
 		})
 }
 
+func (s *Store) UpdateExport(ctx context.Context, export model.Export) (err error) {
+	_, span := s.tracer.Start(ctx, "keboola.go.buffer.configstore.UpdateExport")
+	defer telemetry.EndSpan(span, &err)
+
+	err = s.DeleteExport(ctx, export.ExportKey)
+	if err != nil {
+		return err
+	}
+	err = s.CreateExport(ctx, export)
+	if err != nil {
+		return err
+	}
```

Comment:
Nemělo by tohle proběhnout v transakci? Vím jen tak matně že je etcd má nějaký omezený a že jste ohledně nich teď měli nějaký PR, ale to je tak všechno. 🙂 Tak nevím jestli jste to s Michalem nějak řešili.

Reply:
Preco to robime takto? :slightly_smiling_face: 

Nie je to atomicke, nemusi sa to mazat, staci kluc prepisat .... napr. `createExportOp` moze mat nejaky `bool flag` `Put/PutIfNotExists`, alebo spravit nejaky 3 `Put....` nad op. ktory bude mat flag.

Alebo `Put(key)` , a `Put(WithIfNotExists(bool))`

Reply:
To plati pre vsetky updates.

Reply:
Neděláme to takhle, upravilo se to v dalším commitu https://github.com/keboola/keboola-as-code/pull/1037/commits/dfc50c3038b4f1dd759ffc9464465873ea0fbae9

Reply:
super :+1: